### PR TITLE
[9.x][vi] Completed missing translations 

### PR DESCRIPTION
**Pull request by dinhquochan**

### CODE REVIEW

File: app/main/Constants/Referents.php [hunk 1]
```diff
@@ -32,6 +32,8 @@ class Referents
 
     public const uk = ['MrAlKuz'];
 
+    public const vi = ['dinhquochan'];
```
Comment (andrey-helldar) [inline app/main/Constants/Referents.php:35]: @caouecs, we have a new person to join our team 😊

**Comment by andrey-helldar** [comment]
@dinhquochan, In your language, the names of the countries coincide with English or do you not translate them?

File: app/main/Constants/Referents.php (hunk 1 above)
Comment (caouecs) [reply app/main/Constants/Referents.php:35]: done

**Comment by caouecs** [comment]
Thank you

**Comment by dinhquochan** [comment]
> @dinhquochan, In your language, the names of the countries coincide with English or do you not translate them?

I prefer to use original name but will translate this in feature!

**Pull request merged**